src/components/core/enums.ts

```diff
@@ -21,4 +21,5 @@ export enum DisplayLayout {
 export enum FeatureFlag {
   GENERAL_MEMBERSHIP_MINT = 'GENERAL_MEMBERSHIP_MINT',
   SINGLE_COLLECTION = 'SINGLE_COLLECTION',
+  OPENGRAPH_IMAGES = 'OPENGRAPH_IMAGES',
 }
```

Comment:
Feels a little clunky that I have to import this enum separately from the `isFeatureFlagEnabled`, might be worth either moving this enum to `featureFlags.ts` or allowing the `isFeatureFlagEnabled` to accept a string key and lean on TS to ensure that it's a valid enum key.

Reply:
yeahh agreed, string key probably cleanest